src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -181,19 +182,19 @@ We used a `Docker fork environment` for all the Machine Learning tasks. *activee
 
 NOTE: The use of docker containers is recommended as that way other tasks will not be affected by change. Docker containers provide isolation so that the host machine’s software stays the same. More details available on https://www.activeeon.com/public_content/documentation/latest/user/ProActiveUserGuide.html#_fork_environment[Proactive User Guide^]
 
-=== Publish a ML task
+=== Publish a ML Task
 
-The `Catalog` menu provides the possibility for a user to publish new created or/and update tasks inside *Machine Learning Bucket*, you need just to click on `Catalog` Menu then `Publish current Workflow to the Catalog`.
+The `Catalog` menu provides the possibility for a user to publish newly created or/and update tasks inside *Machine Learning Bucket*, you need just to click on `Catalog` Menu then `Publish current Workflow to the Catalog`.
 Choose `machine-leaning` Bucket to store your new added workflow on it.
```

Comment:
new --> newly

Reply:
@luispinedamx  the correct is "new" or "newly" ? because the old version was "new" and I modified to "newly" .

Reply:
@carolinepacheco it is "newly", it was just missing in line 188:
"Choose `machine-leaning` Bucket to store your **newly** added workflow on it."